.github/workflows/build.yml

```diff
@@ -21,6 +21,17 @@ env:
   INSTALL_PREFIX: ${{ github.workspace }}/nvim-install
 
 jobs:
+  wasmtime:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/setup
+      - run: |
+          cmake -S cmake.deps --preset ci -D ENABLE_WASM=ON
+          cmake --build .deps
+          cmake --preset ci -D ENABLE_WASM=ON
```

Comment:
is this intentionally not ENABLE_WASMTIME ? same question for `cmake.deps/CMakePresets.json`

Reply:
It's wrong. Thanks for the heads up.